snmp/datadog_checks/snmp/data/profiles/cisco-legacy-wlc.yaml

```diff
@@ -0,0 +1,51 @@
+# Profile for Cisco WLC
+
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+  - _generic-tcp.yaml
+  - _generic-udp.yaml
+  - _cisco-wlc.yaml
+
+device:
+  vendor: "cisco"
+
+sysobjectid:
+  - 1.3.6.1.4.1.9.1.818  # ciscoNMWLCE
+  - 1.3.6.1.4.1.9.1.828  # ciscoAirWlc2106K9
+  - 1.3.6.1.4.1.9.1.1069 # cisco5500Wlc
+  - 1.3.6.1.4.1.9.1.1295 # cisco7500Wlc
+  - 1.3.6.1.4.1.9.1.1615 # cisco8500WLC
+  - 1.3.6.1.4.1.9.1.1631 # ciscoVirtualWlc
+  - 1.3.6.1.4.1.9.1.1645 # cisco5760wlc
+  - 1.3.6.1.4.1.9.1.1926 # ciscoWlcCt5508K9
+  - 1.3.6.1.4.1.9.1.1927 # ciscoWlcCt2504K9
+  - 1.3.6.1.4.1.9.1.2170 # cisco5520WLC
+  - 1.3.6.1.4.1.9.1.2171 # cisco8540Wlc
+  - 1.3.6.1.4.1.9.1.2427 # cisco3504WLC
+
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "cisco"
+      serial_number:
+        symbol:
+          OID: 1.3.6.1.4.1.14179.1.1.1.4
+          name: agentInventorySerialNumber
+
+metrics:
+  - MIB: AIRESPACE-SWITCHING-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.14179.1.1.5.1
+      name: cpu.usage
+  - MIB: AIRESPACE-SWITCHING-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.14179.1.1.5.3
+      scale_factor: 1000
+      name: memory.free
+  - MIB: AIRESPACE-SWITCHING-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.14179.1.1.5.2
+      scale_factor: 1000
+      name: memory.total
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Could you add the symbol name as comment for convenience ? :)

Example:
```
    symbol:
      OID: 1.3.6.1.4.1.14179.1.1.5.1 # agentCurrentCPUUtilization
      scale_factor: 1000
      name: memory.free
```